### PR TITLE
Documentation fix

### PR DESCRIPTION
**Pull request by joecharles33**
Added a one liner command to create the pull-secret secret in the openshift-config namespace

### CODE REVIEW

**Comment by andykrohg** [comment]
Hmm, I'm not sure this is easier or faster than the instructions as written. Did you try the original method and run into problems, @joecharles33?

**Comment by joecharles33** [comment]
@andykrohg I did try the original method and ran into issues. That's why I submitted the pull request.

File: README.adoc [hunk 1]
```diff
@@ -76,24 +76,16 @@ oc delete limitrange --all -n devsecops
...
-      metadata:
-        name: pull-secret
-        namespace: openshift-config
-      data:
-        .dockerconfigjson: $SECRET
-      type: kubernetes.io/dockerconfigjson
-    EOF
-  ----
+  
+  oc create secret generic pull-secret --from-file=.dockerconfigjson=</path/to/.docker/config.json> --type=kubernetes.io/dockerconfigjson
```
Comment (andykrohg) [inline README.adoc:85]: Can we add `-n openshift-config`?

**Review by andykrohg — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 1 above)
Comment (joecharles33) [reply README.adoc:85]: yes we can.

Comment (joecharles33) [reply README.adoc]: i added it.

**Review by andykrohg — APPROVED** [review]

**Pull request merged**